src/Tuttifrutti/Cache/Storage/Postgres.hs

```diff
@@ -0,0 +1,158 @@
+{-# LANGUAGE OverloadedStrings #-}
+-- | A PostgreSQL backend for 'Tuttifrutti.Cache.Storage' interface:
+--
+--   >>> import Data.Time
+--   >>> import Tuttifrutti.Prelude
+--   >>> import qualified Tuttifrutti.Log.Handle as Log
+--   >>> import qualified Tuttifrutti.Cache.Storage as Storage
+--   >>> import qualified Tuttifrutti.Cache.Storage.Postgres as Storage.Postgres
+--   >>>
+--   >>> logHandle <- Log.newStdoutHandle "tuttifrutti" Log.devMessage
+---  >>> let connect = Postgres.connect logHandle PG.defaultConnectInfo Postgres.defaultRetryPolicy
+--   >>> pool <- Pool.createPool connect PG.close Pool.defaultConfig
```

Comment:
Here you use `Postgres`, `Pool`, `PG`, but don't import them above, do they come from `Tuttifrutti.Prelude`?

Also, should we run doctests on this stuff?

Reply:
Yes, there were some misimports, now fixed and we have doctests :tada: